src/main/webapp/dev/js/common/questionMcq.js

```diff
@@ -37,13 +56,14 @@ function addMcqOption(questionNum) {
 function removeMcqOption(index, questionNum) {
     const questionId = `#form_editquestion-${questionNum}`;
 
-    const $thisRow = $(`#mcqOptionRow-${index}-${questionNum}`);
+    const $thisRow = $(`.mcqChoiceRow-${index}-${questionNum}`);
```

Comment:
I have used class so that I have a way to manipulate a specific choice and it's corresponding weight cell together. Each choice and it's corresponding weight cell falls under the same class.

Reply:
You can also put an `id` on the enclosing `row` of the option and weight fragments.

Reply:
Personally I prefer using `id` to get the choice and weight here since this class name is kind of inconsistent with the rest of the code, where `index` and `questionNum` are used in `id`s only.

Reply:
> where index and questionNum are used in ids only.
@sukanta-27 Yes. I think HTML class is not designed to be used like that

Reply:
@xpdavid @bqnguyen94 Okay. will use ids here.